pkgs/circuits/halo2-secp256k1-wasm/src/draft/mod.rs

```diff
@@ -119,7 +119,13 @@ mod tests {
         let k_inv = k.invert().unwrap();
 
         // Calculate `r`
+<<<<<<< HEAD
         let r_point = Secp256k1Affine::from(G * k).coordinates().unwrap();
+=======
+        let r_point = Secp256k1Affine::from(G * k)
+            .coordinates()
+            .unwrap();
+>>>>>>> 49a75ad (feat: halo2lib secp256k1 recover pk)
```

Comment:
Fix your merge conflicts characteres

Reply:
or maybe this `draft` folder was a temp folder you didn't want to version control?